src/kafka/subscribers/messages.ts

```diff
@@ -30,7 +30,13 @@ export const fromKafkaToConsumedMessage = async (message: KafkaMessage): Promise
 export const getMessagesFromRedis = async (subscriberId: string): Promise<ConsumedMessage[]> => {
   const serializedMessages = await getRedisClient().lRange(toMessagesKey(subscriberId), 0, -1);
   const messages = serializedMessages.map(
-    (message: string) => JSON.parse(message) as ConsumedMessage,
+    (message: string) => {
```

Comment:
Maybe we should name it `serializedMessageObject: string`
and then:
```
const messageObject = JSON.parse(serializedMessageObject);
```

Just for better understanding

Reply:
Good point.